packages/astro/src/core/add/index.ts

```diff
@@ -727,6 +739,128 @@ export async function validateIntegrations(integrations: string[]): Promise<Inte
 	}
 }
 
+async function updateTSConfig(
+	cwd = process.cwd(),
+	logging: LogOptions,
+	integrationsInfo: IntegrationInfo[],
+	flags: yargs.Arguments
+): Promise<UpdateResult> {
+	const frameworkWithTSSettings = ['solid-js', 'preact', 'react', 'vue'] as const;
```

Comment:
a nit since the array is so small, but using a Set is generally better when used as a lookup like this is doing.